packages/dd-trace/src/opentracing/span.js

```diff
@@ -278,6 +278,10 @@ class DatadogSpan {
     this._processor.process(this)
   }
 
+  sample () {
```

Comment:
This adds new API surface area so it would need:

(1) its own test
(2) a corresponding PR against dd-trace-api-js
(3) the appropriate changes in the dd-trace-api integration in this repo

Reply:
I also think adding an API like that should be independent from this PR. There is already a mechanism in place to sample before propagation, so we could use the same mechanism instead of calling `sample` directly.

Reply:
@rochdev I'm sorry, but which mechanism do you have in mind? The reason I added this API is because I didn't find a way to do it from all the DB plugins.